performance/implementations/glamor/Box/index.js

```diff
@@ -6,12 +6,7 @@ import View from '../View';
 const Box = ({ color, fixed = false, layout = 'column', outer = false, ...other }) => (
   <View
     {...other}
-    style={[
-      styles[`color${color}`],
-      fixed && styles.fixed,
-      layout === 'row' && styles.row,
-      outer && styles.outer
-    ]}
+    style={[styles[`color${color}`], fixed && styles.fixed, layout === 'row' && styles.row, outer && styles.outer]}
```

Comment:
prettier relies heavily on column count for this, which works well for 80 columns. Since you pushed it to 120, you are going to be able to cram a lot more inside.

One solution that we are considering is to expand the array if it was expanded in the original source. We already do this for objects and it solved a ton of hard problems to algorithmically decide on.

https://github.com/prettier/prettier/issues/817#issuecomment-283075178